### ISSUE TITLE
Compile Issue with Error Message

### ISSUE THREAD

**Issue by SaraJo** [Jewelbots/arduino-library#1]
When a sketch is uploaded, an error message occurs that is not an actual error. This is confusing for the user. The message is

`Failed to execute script __main__`

Explanation from @davison-sm here:

The tool is here:  https://github.com/NordicSemiconductor/pc-nrfutil

There are multiple branches with different versions.  Master is the updated tool for SDK 12 and will not work for us as we haven't upgraded to 12 yet.  The branch that will work for us is "0_5_2".

The code works fine when installed normally in the system using "python setup.py install".  This nrfutil tool runs perfectly and doesn't give the error message.  But it is installed on the system using links, so the executable can't be distributed on our github.

The standard method of making a distributable executable would be "pyinstaller --onefile setup.py".  The installer builds correctly, but gives an error about a broken import module command when it is run.

To make the executable that we are using now, run pyinstaller on the main.py file in the nordicsemi folder.  This executable runs but has the annoying error message.

**Comment by rwaldron** [comment]
I tried to follow these directions, but something isn't connecting. Can you do a step-by-step list?

**Comment by SaraJo** [comment]
Seen in issue #3.

**Comment by oldanderw** [comment]
Hello I am new to JB but I been trying to look into this from day 2 and I have found something that seems consistent that may help in find out what is causing that and I apologize if I am information that you already know, I tried looking around to make sure that I was not doing that. before I get started, I want tried testing on friendship and solo lib. so that first I tried was booting into my windows machine(macbook pro user here) and I did not get this error at all either by checking the sketch and/or uploading to the my sketch to my JB and then I tried also doing the same thing with Linux since I did see that it can run on Linux and again there were no errors what so ever and the only time I ever saw `Failed to execute script __main__` error message was using my macbook pro. I also tried google this because I hope that this isn't the very first time this has happened on Arduino and hopefully it was reported but I got no luck other than finding https://github.com/esp8266/Arduino/issues/3 and saw in the error log that the __main__.cpp file was failing but I have no idea if this is related at all just from looking around. 

This is what I have found so far, I have that I have help little bit and not hurt. I am still trying to look into this.

EDIT: also I noticed that the error appears at the beginning  and the ending of the log when checking or upload sketches

**Comment by SaraJo** [comment]
Thank you, @oldanderw, this will be helpful when we are working on this.

**Comment by cferraro** [comment]
Hello, any updates on this issue?

**Comment by SaraJo** [comment]
No, we dug into it and it turned out to be an issue deep within the firmware that would be difficult to change. We updated the documentation to reflect this as an expected message.

**Comment by cferraro** [comment]
Thanks for your reply. It seem it's ok to ignore the error, so will ignore. Thx
